docs/python_docs/python/tutorials/deploy/inference/image_classification_jetson.md

```diff
@@ -0,0 +1,130 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Image Classication using pretrained ResNet-50 model on Jetson module
+
+This tutorial shows how to install latest MXNet v1.6 with Jetson support and use it to deploy a pre-trained MXNet model for image classification on a Jetson module.
+
+## What's in this tutorial?
+
+This tutorial shows how to:
+
+1. Install MXNet v1.6 with Jetson support along with its dependencies
+
+2. Deploy a pre-trained MXNet model for image classifcation on a Jetson module
+
+### Who's this tutorial for?
+
+This tutorial would benefit developers working on any Jetson module implementing a deep learning application. It assumes that readers have a Jetson module setup, are familiar with the Jetson working environment and are somewhat familiar with deep learning using MXNet.
+
+### How to use this tutorial?
+
+To follow this tutorial, you need to setup a [Jetson module](https://developer.nvidia.com/embedded/develop/hardware) and install latest [Jetpack 4.4](https://docs.nvidia.com/jetson/jetpack/release-notes/) using NVIDIA [SDK manager](https://developer.nvidia.com/nvidia-sdk-manager).
+
+All instructions described in this tutorial can be executed on the any Jetson module directly or via SSH.
+
+## Prerequisites
+
+To complete this tutorial, you will need:
+
+* A Jetson module with Jetpack 4.4 installed
+* [Swapfile](https://help.ubuntu.com/community/SwapFaq) installed (in case of Jetson Nano) for additional memory
```

Comment:
I would suggest you just come out and say this tutorial is for Jetson Nano specifically. Until you verify and go through the steps for other boards dont want to give the impression that it will "just work".

Reply:
> I would suggest you just come out and say this tutorial is for Jetson Nano specifically. Until you verify and go through the steps for other boards dont want to give the impression that it will "just work".

This tutorial has been tested on the two Jetson modules - Nano and Xavier AGX. I'll specify the device names to be more specific.

> Also, you should state that all the setup steps have been completed and you have an ssh connection or directly have a shell open on the board. The next steps dive in to giving commands to execute.

I'll add this information.